modules/Template/PortalDataset.yaml

```diff
@@ -20,12 +20,16 @@ classes:
     # - doi (if minted)
     # - createdBy # user id/handle, creator allows setting name more explicitly for citations
     # - datasetItemCount
-    # - datasetSizeInBytes 
+    # - datasetSizeInBytes
     # - publisher # should be autoset
-      title:
-        title: Title
+      name:
```

Comment:
Originally, title was used to conform to https://www.w3.org/TR/vocab-dcat-3/#Property:resource_title. I think using `name` means we won't have as much control also since in Synapse it's reserved and there are character restrictions on names (so you can't use "/", which can appear a lot). Will there be cases when it might be frustrating to not be able to name your dataset "Data from Phase 1a/b trial" or "Data from NF1+/- model mouse"?  (alternative, we can ask platform if it's possible to do do away with this)

Reply:
I guess we can use `alternateName`